### PR TITLE
Check whether given number is palindrome

### PR DESCRIPTION
**Pull request by twentyonedot**
Added a new basic problem: 'Check whether the given number is palindrome or not?'

### CODE REVIEW

File: Palindrome.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Pallindrome {
+
+    public static void main(String[] args) {
+        int n=1231;
```
Comment (codec-akash) [inline Palindrome.java]: Can you use scanner to take the user input, If not its all good. this still works.

**Review by codec-akash — COMMENTED** [review]
Can you add a scanner to take user input if possible, else this is also good.

File: Palindrome.java (hunk 1 above)
Comment (twentyonedot) [reply Palindrome.java]: Yes definitely. I will add that.

**Comment by twentyonedot** [comment]
Done added the scanner.

**Comment by codec-akash** [comment]
> Done added the scanner.

Thank you.

**Pull request merged**